tests/testQuestionParams.js

```diff
@@ -0,0 +1,336 @@
+    it('should load successfully', function (callback) {
+      request(addNumbers1_assessment.url, function (error, response, body) {
+        if (error) {
+          return callback(error);
+        }
+        if (response.statusCode !== 200) {
+          return callback(new Error('bad status: ' + response.statusCode));
+        }
+        page = body;
+        callback(null);
+      });
+    });
+
+    it('should parse', function () {
+      locals.$ = cheerio.load(page);
+    });
+
+    it('should have a range set by the course instance questionParams', function () {
+      elemList = locals.$('span:contains("[0, 10]")');
+      assert.lengthOf(elemList, 1);
+    });
+  });
+
+  describe('3. Assessment question inherits parameters from zone', function () {
+    it('should load successfully', function (callback) {
+      request(addNumbers2.url, function (error, response, body) {
+        if (error) {
+          return callback(error);
+        }
+        if (response.statusCode !== 200) {
+          return callback(new Error('bad status: ' + response.statusCode));
+        }
+        page = body;
+        callback(null);
+      });
+    });
+
+    it('should parse', function () {
+      locals.$ = cheerio.load(page);
+    });
+
+    it('should have a range set by the zone questionParams', function () {
+      elemList = locals.$('span:contains("[0, 10]")');
+      assert.lengthOf(elemList, 1);
+    });
+  });
+
+  describe('4. Assessment question inherits parameters from alternatives group', function () {
+    it('should load successfully', function (callback) {
+      request(addNumbers3.url, function (error, response, body) {
+        if (error) {
+          return callback(error);
+        }
+        if (response.statusCode !== 200) {
+          return callback(new Error('bad status: ' + response.statusCode));
+        }
+        page = body;
+        callback(null);
+      });
+    });
+
+    it('should parse', function () {
+      locals.$ = cheerio.load(page);
+    });
+
+    it('should have a range set by the alternatives group questionParams', function () {
+      elemList = locals.$('span:contains("[10, 20]")');
+      assert.lengthOf(elemList, 1);
+    });
+  });
+
+  describe('5. Assessment question inherits parameters from assessment question', function () {
+    it('should load successfully', function (callback) {
+      request(addNumbers4.url, function (error, response, body) {
+        if (error) {
+          return callback(error);
+        }
+        if (response.statusCode !== 200) {
+          return callback(new Error('bad status: ' + response.statusCode));
+        }
+        page = body;
+        callback(null);
+      });
+    });
+
+    it('should parse', function () {
+      locals.$ = cheerio.load(page);
+    });
+
+    it('should have a range set by the assessment question questionParams', function () {
+      elemList = locals.$('span:contains("[20, 30]")');
+      assert.lengthOf(elemList, 1);
+    });
```

Comment:
After getting to the end of this file it's looking like this is just a copy/paste of existing assessment tests, so take my comments here with a grain of salt. That said, while it's unfortunate that old tests aren't written how we'd like, I'd like new tests like this to follow best practices (use async/await, `it` blocks should contain assertions and not just logic, etc.)

We should really refactor our tests so we can share boilerplate like `startAssessment` and such, but that's outside what I'd expect you to do for this PR.